protocol-council-charter.md

```diff
@@ -0,0 +1,301 @@
+meeting of the Council at the written request of a majority of the
+members.
+
+Any Council member that is a permanent resident of a non-U.S.
+jurisdiction may act as the executor on any Council-approved Transaction
+(as defined below) effected through the Council's multi-signature wallet
+("**Multisig**") used to effect deployments, upgrades, and/or
+administrative actions (each a "**Transaction**") with respect to the
+EigenLayer Core Contracts identified in Exhibit A ("**EigenLayer Core
+Contracts**"). The Chair shall coordinate the Council's approval and
+execution of any Multisig Transaction only if the Council has previously
+approved the Multisig Transaction and provided notice to the Board. In
+the event that a Council-approved Multisig Transaction is not executed
+timely, the Chair shall have the authority to instruct any Council
+member to participate as approver or executor on the Multisig (each, a
+"**Signer**"), in accordance with this Charter, the Foundation's
+constitutional documents, and bylaws.
+
+#### **5. Operations**
+
+The Council will meet with such frequency as the Chair or it may
+determine advisable, in accordance with the ELIP process and
+requirements of EigenGov. Quorum for Council meetings shall be three (3)
+Council members present in person or virtually save where there are only
+two (2) Council members in place, in which case quorum for Council
+meetings shall be (2) Council members present in person or virtually.
+
+The Council's approval of any security upgrade, ELIP, or other Council
+matter shall require a majority of the votes of the Council members
+present. In the event the Chair declares an emergency and provides
+immediate notice to the Board, the Chair may determine to treat any
+Signer's participation on a Transaction as an affirmative vote for the
+approved or executed Transaction. Each member shall be a key holder with
+respect to the Council's Multisig Wallet. The threshold Signers required
+for Transactions may be as few as three (3) members, unless the Chair
+and/or Council determine to require a higher number of Signers. In the
+event that the Council has fewer than three (3) members, the threshold
+Signers required for Transactions shall be at least the number of
+Council members. The Chair may appoint Foundation personnel to serve as
+temporary Signers on terms and conditions decided by the Chair.   
+
+The Council will regularly report to the Board on the Council's
+actions.  
+
+The Council may invite to its meetings non-Council members, including
+any persons the Council deems appropriate in order to carry out its
+responsibilities.  
+
+#### **6. Conflicts of Interest**
+
+Prior to exercising voting and/or Multisig authority, the Council must
+review all relevant facts and circumstances among the members to
+determine whether a material conflict of interest may arise due to a
+member's [family, social, financial, or professional]{.mark} interests
```

Comment:
these {.mark} are not displaying correctly

Reply:
fixed